internal/pkg/cli/storage_init.go

```diff
@@ -399,13 +399,20 @@ func (o *initStorageOpts) askDynamoLSIConfig() error {
 		return fmt.Errorf("confirm add alternate sort key: %w", err)
 	}
 	for {
-		if !moreLSI {
-			break
-		}
 		if len(o.lsiSorts) > 5 {
 			log.Infoln("You may not specify more than 5 alternate sort keys. Continuing...")
+			moreLSI = false
+		}
+		// This will execute last in the loop if moreLSI is set to false by any confirm prompts.
+		if !moreLSI {
+			if len(o.lsiSorts) == 0 {
+				o.noLsi = true
+			} else {
+				o.noLsi = false
+			}
 			break
 		}
```

Comment:
can we have:
```suggestion
		if !moreLSI {
			o.noLsi = len(o.lsiSorts) == 0
			return nil
		}
```

Reply:
also can we update `noLsi` to `noLSI`

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Done.